en/docs/guides/access-delegation/oauth-pushed-authorization-requests.md

```diff
@@ -0,0 +1,79 @@
+# Pushed Authorization Requests (PAR)
+
+Pushed Authorization Request (PAR) is an OAuth 2.0 specification that secures the way of initiating an authorization flow.
+
+The PAR specification defines the `/par` endpoint in the authorization server, and enables an OAuth client to push the payload of an authorization request directly to this endpoint. The endpoint in response, will return `request_uri`, a one-time use code which acts as a reference to the registered authorization request.
+
+The client authentication rules defined for token endpoint requests, including the relevant authentication methods, also apply to the `/par` endpoint.
+
+When making the authorization request to the authorization endpoint, it is sufficient to send the `request_uri` instead of sending the complete authorization payload.
```

Comment:
```suggestion
When making the authorization request to the authorization endpoint, it is sufficient to send the `request_uri` and `client_id`, instead of sending the complete authorization payload.
```